src/uct/ib/ud/base/ud_iface.h

```diff
@@ -381,9 +385,27 @@ uct_ud_iface_get_async_time(uct_ud_iface_t *iface)
     return iface->super.super.worker->async->last_wakeup;
 }
 
+static UCS_F_ALWAYS_INLINE ucs_time_t
+uct_ud_iface_get_time(uct_ud_iface_t *iface)
+{
+    return ucs_get_time();
+}
+
+static UCS_F_ALWAYS_INLINE void
+uct_ud_iface_twheel_sweep(uct_ud_iface_t *iface)
+{
+    if (((iface->tx.skip_sweep++ % UCT_UD_SKIP_SWEEP) == 0) &&
+        !iface->tx.disable_sweep &&
```

Comment:
let's make the check "disable_sweep" last, since it's for debug, to make less checks when ep's are all empty

Reply:
maybe make the check when adding timer instead?

Reply:
you mean to not add if sweep is disabled?
so the variable name should be disable_timer rather than disable_sweep